src/components/DataTable.tsx

```diff
@@ -126,6 +141,18 @@ export function DataTable<T extends object>(props: Props<T>) {
     new URL(endpoint(props.endpoint))
   );
 
+  const tableKey = useMemo(() => `${location.pathname}${props.endpoint}`, []);
```

Comment:
Please be careful with the overuse of useMemo. If used for smaller chunked values, like this one, it can actually _hurt_ performance.

Reply:
@beganovich Ah, okay. Thank you for providing me with this previously unknown information. I will be careful when using it and will only apply it in appropriate contexts. It has been updated, and we now use plain values.